src/containerapp/azext_containerapp/_validators.py

```diff
@@ -215,3 +220,49 @@ def validate_timeout_in_seconds(cmd, namespace):
     if timeout_in_seconds is not None:
         if timeout_in_seconds < 0 or timeout_in_seconds > 60:
             raise ValidationError("timeout in seconds must be in range [0, 60].")
+
+
+def validate_debug(cmd, namespace):

```

Comment:
looks like we are making separate calls to set defaults and validate but is it possible to merge all these calls? because if we set default values and we should be pretty confident that they are valid